app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    cp, old_file, new_file = command.split()
+
+    if cp != "cp":
+        raise ValueError("The command need to start with 'cp'")
+
+    if old_file != new_file:
```

Comment:
Code Efficiency Issue: The code checks if the old file name is different from the new file name, but it does not check if the old file actually exists. You should add a check for this.